base_external/configs/aesd_rpi_defconfig

```diff
@@ -15,6 +19,27 @@ BR2_LINUX_KERNEL_DTS_SUPPORT=y
 BR2_LINUX_KERNEL_INTREE_DTS_NAME="bcm2710-rpi-3-b bcm2710-rpi-3-b-plus bcm2710-rpi-cm3"
 BR2_LINUX_KERNEL_NEEDS_HOST_OPENSSL=y
 BR2_PACKAGE_RPI_FIRMWARE=y
+BR2_PACKAGE_RPI_WIFI_FIRMWARE=y
+BR2_PACKAGE_PYTHON=y
+BR2_PACKAGE_PYTHON_HASHLIB=y
+BR2_PACKAGE_PYTHON_PAHO_MQTT=y
+BR2_PACKAGE_DROPBEAR=y
+BR2_PACKAGE_DROPBEAR_DISABLE_REVERSEDNS=y
+BR2_PACKAGE_MACCHANGER=y
+BR2_PACKAGE_MOSQUITTO=y
+BR2_PACKAGE_WIRELESS_TOOLS=y
```

Comment:
@Sankalppund I don't think packages wireless/wpa are required, correct?  If so can you remove?

Reply:
I have added those packages to setup WiFi which will be required for MQTT communication.

Reply:
I think you are missing some wifi support necessary content, so I'd rather just make this PR about support of mqtt via python.

Reply:
I have made the necessary changes and now this PR is supporting MQTT via python.